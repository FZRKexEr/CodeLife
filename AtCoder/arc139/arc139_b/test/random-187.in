100
68 6 15 3 6 7
90 17 1 4 12 18
25 6 10 20 19 7
73 10 13 2 3 2
5 14 1 2 19 19
96 12 7 8 10 7
82 17 7 18 3 3
23 4 19 9 10 4
89 6 8 6 12 16
66 4 16 14 17 17
40 6 19 2 11 2
31 5 19 1 20 12
30 13 1 15 10 2
24 3 4 1 13 8
87 11 17 19 15 6
44 9 16 2 12 4
71 18 14 20 11 5
63 14 2 13 14 15
86 5 15 3 18 9
40 11 11 8 4 7
55 8 9 9 13 11
26 9 10 9 16 15
98 2 19 11 3 17
26 13 9 5 18 2
58 19 4 5 1 19
48 19 9 3 12 5
62 3 5 18 17 1
90 6 16 15 4 4
71 14 14 4 6 17
96 4 6 12 10 14
71 20 11 12 9 8
63 18 3 10 12 6
92 20 20 6 16 7
77 13 17 12 9 17
33 2 4 10 17 14
83 13 13 15 3 14
90 6 17 13 4 14
63 17 6 7 2 4
80 19 2 13 10 20
63 16 5 14 15 20
73 10 12 19 13 11
11 5 1 19 18 15
23 6 14 3 8 7
98 6 2 9 16 9
72 3 16 5 15 2
74 14 2 5 3 12
64 20 17 4 8 3
51 15 5 13 14 17
91 6 6 20 5 17
32 7 3 2 14 17
66 20 4 1 14 10
16 6 2 16 17 13
74 2 19 8 16 18
99 16 13 14 1 10
50 7 12 2 1 6
21 10 15 2 19 17
64 15 12 3 6 14
15 2 9 17 13 2
93 14 17 15 7 14
68 9 18 18 17 2
28 4 10 20 10 3
1 19 12 9 20 3
16 2 17 6 20 7
30 1 13 5 15 3
40 10 13 19 2 2
32 7 18 15 6 18
10 9 16 11 19 11
90 2 3 9 16 4
70 2 14 8 1 4
52 11 1 15 12 9
78 13 12 1 15 1
9 1 3 9 6 9
11 5 1 11 17 9
92 3 20 11 17 17
92 4 16 14 9 9
15 16 1 13 13 11
58 17 18 18 19 15
13 20 2 17 13 14
95 17 10 19 10 16
73 6 12 18 15 15
78 12 16 20 18 17
25 20 9 2 4 2
65 13 13 9 16 3
14 20 10 10 18 8
56 11 12 14 3 6
82 13 10 20 8 10
86 12 20 3 4 20
9 8 3 17 14 18
30 17 6 9 11 8
32 7 12 17 6 12
83 14 5 11 14 8
3 18 9 5 4 1
29 9 11 4 13 18
58 12 5 19 17 10
76 7 18 1 8 16
86 9 11 11 3 12
97 6 2 7 2 20
79 11 6 4 14 6
81 1 17 14 16 2
85 2 1 3 9 10
