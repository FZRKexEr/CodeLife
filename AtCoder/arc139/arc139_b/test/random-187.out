40
80
96
16
10
80
18
14
182
96
8
24
28
24
68
26
45
114
78
68
75
84
99
36
8
32
48
40
34
224
35
54
100
69
122
93
60
14
80
105
85
51
24
265
48
35
40
72
95
64
48
60
190
48
9
29
36
41
63
113
26
9
96
26
8
30
30
120
20
120
12
27
43
136
72
165
117
101
66
108
126
14
15
48
18
62
34
31
36
47
116
15
44
88
68
56
39
82
190
255
